### PR TITLE
Cut down on javadoc warnings about missing tags.

### PR DESCRIPTION
**Pull request by wsargent**
I was on a plane, so I went through all the javadoc warnings and started adding javadoc tags.

### CODE REVIEW

File: framework/src/play-test/src/main/java/play/test/Helpers.java [hunk 1]
```diff
@@ -461,28 +497,34 @@ public static void running(TestServer server, WebDriver webDriver, final Consume
...
+     *
+     * @return the test browser.
      */
     public static TestBrowser testBrowser() {
         return testBrowser(HTMLUNIT);
     }
 
     /**
      * Creates a Test Browser.
+     *  @return the test browser.
```
Comment (schmitch) [inline framework/src/play-test/src/main/java/play/test/Helpers.java]: guess this should be:
```
*
* @return.....
```

File: framework/src/play-test/src/main/java/play/test/Helpers.java [hunk 2]
```diff
@@ -461,28 +497,34 @@ public static void running(TestServer server, WebDriver webDriver, final Consume
...
      */
     public static TestBrowser testBrowser(int port) {
         return testBrowser(HTMLUNIT, port);
     }
 
     /**
      * Creates a Test Browser.
+     *  @return the test browser.
```
Comment (schmitch) [inline framework/src/play-test/src/main/java/play/test/Helpers.java]: guess this should be:
```
*
* @return.....
```

Comment (schmitch) [inline framework/src/play-test/src/main/java/play/test/Helpers.java]: like above

File: framework/src/play-test/src/main/java/play/test/Helpers.java [hunk 3]
```diff
@@ -496,13 +538,15 @@ public static TestBrowser testBrowser(Class<? extends WebDriver> webDriver, int
...
```
Comment (schmitch) [inline framework/src/play-test/src/main/java/play/test/Helpers.java]: like above

File: framework/src/play-test/src/main/java/play/test/Helpers.java [hunk 4]
```diff
@@ -496,13 +538,15 @@ public static TestBrowser testBrowser(Class<? extends WebDriver> webDriver, int
...
      */
     public static TestBrowser testBrowser(WebDriver of, int port) {
         return new TestBrowser(of, "http://localhost:" + port);
     }
 
     /**
      * Creates a Test Browser.
+     *  @return the test browser.
```
Comment (schmitch) [inline framework/src/play-test/src/main/java/play/test/Helpers.java]: like above

File: framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java [hunk 5]
```diff
@@ -25,8 +25,11 @@
 
 /**
  * Default implementation of the http error handler.
- *
+ * <p>
```
Comment (schmitch) [inline framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java]: shouldn't this be closed somewhere?

File: framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java [hunk 6]
```diff
@@ -75,9 +82,12 @@ public DefaultHttpErrorHandler(
 
     /**
      * Invoked when a client makes a bad request.
+     * <p>
```
Comment (schmitch) [inline framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java]: like above

File: framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java [hunk 7]
```diff
@@ -87,21 +97,29 @@ public DefaultHttpErrorHandler(
 
     /**
      * Invoked when a client makes a request that was forbidden.
+     * <p>
```
Comment (schmitch) [inline framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java]: like above

File: framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java [hunk 8]
```diff
@@ -128,13 +149,14 @@ public DefaultHttpErrorHandler(
 
     /**
      * Invoked when a server error occurs.
-     *
+     * <p>
```
Comment (schmitch) [inline framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java]: like above

File: framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java [hunk 9]
```diff
@@ -157,22 +179,24 @@ public DefaultHttpErrorHandler(
...
-                        usefulException.id, request.method(), request.uri()),
+                usefulException.id, request.method(), request.uri()),
                 usefulException
         );
     }
 
     /**
      * Convert the given exception to an exception that Play can report more information about.
-     *
+     * <p>
```
Comment (schmitch) [inline framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java]: like above

File: framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java [hunk 10]
```diff
@@ -181,9 +205,11 @@ private UsefulException throwableToUsefulException(final Throwable throwable) {
     }
 
     /**
-     * Invoked in dev mode when a server error occurs.
+     * Invoked in dev mode when a server error occurs.  Note that this method is where the URL set by play.editor is used.
+     * <p>
```
Comment (schmitch) [inline framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java]: like above

File: framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java [hunk 11]
```diff
@@ -192,12 +218,16 @@ private UsefulException throwableToUsefulException(final Throwable throwable) {
 
     /**
      * Invoked in prod mode when a server error occurs.
-     *
+     * <p>
+     * The base implementation returns {@code Results.internalServerError} with the content of {@code views.html.defaultpages.error} template.
+     * </p>
+     * <p>
```
Comment (schmitch) [inline framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java]: like above

File: framework/src/play/src/main/java/play/libs/streams/AkkaStreams.java [hunk 12]
```diff
@@ -22,9 +22,19 @@
 
     /**
      * Bypass the given flow using the given splitter function.
-     *
+     * <p>
```
Comment (schmitch) [inline framework/src/play/src/main/java/play/libs/streams/AkkaStreams.java]: doesn't close

File: framework/src/play/src/main/java/play/libs/streams/AkkaStreams.java [hunk 13]
```diff
@@ -22,9 +22,19 @@
...
      * If the splitter function returns Left, they will go through the flow.  If it returns Right, they will bypass the
      * flow.
+     * <p>
```
Comment (schmitch) [inline framework/src/play/src/main/java/play/libs/streams/AkkaStreams.java]: doesn't close

File: framework/src/play/src/main/java/play/libs/streams/AkkaStreams.java [hunk 14]
```diff
@@ -34,12 +44,21 @@
 
     /**
      * Using the given splitter flow, allow messages to bypass a flow.
-     *
+     * <p>
```
Comment (schmitch) [inline framework/src/play/src/main/java/play/libs/streams/AkkaStreams.java]: doesn't close

File: framework/src/play/src/main/java/play/mvc/Result.java [hunk 15]
```diff
@@ -217,13 +222,20 @@ public Cookies cookies() {
         return JavaResultExtractor.getCookies(header);
     }
 
+    /**
+     * Returns a copy of this result with the given cookies.
+     * @param cookies the cookies to add to the result.
```
Comment (schmitch) [inline framework/src/play/src/main/java/play/mvc/Result.java]: new line?

File: framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java (hunk 5 above)
Comment (gmethvin) [inline framework/src/play/src/main/java/play/http/DefaultHttpErrorHandler.java]: In HTML it's okay to omit the `</p>` tag in many cases (see https://www.w3.org/TR/html-markup/p.html#p-tags). It's pretty much standard practice in Javadoc so I think it's fine.

**Comment by gmethvin** [comment]
LGTM besides the minor formatting things mentioned by @schmitch (sans the p tag issue).

**Pull request merged**